src/Editor/CodelensProvider.ts

```diff
@@ -22,7 +22,22 @@ export class CodelensProvider implements vscode.CodeLensProvider {
   public provideCodeLenses(document: vscode.TextDocument, token: vscode.CancellationToken):
       vscode.CodeLens[]|Thenable<vscode.CodeLens[]> {
     let codeLenses: vscode.CodeLens[] = [];
-    // TODO Add more
+
+    if (vscode.workspace.getConfiguration('one-vscode').get('enableCodeLens', true)) {
+      let regex = new RegExp(/(.+)/g);
+      let activatedEditorText = document.getText();
```

Comment:
(optional) As I don't have much experience about this...
And as `.cfg` file won't be that huge, but just to be safe...
Can you test with, say 10,000 lines of codes, and check any performance drop?

Reply:
If this behaves like other extension does, it's OK with me.

Reply:
I wrote one-build through snippet and copied nearly 10,000 lines of the content to create a cfg file and obtained the following results.

1,0075 lines (including empty lines) : 115, 146, 86, 109, 120 ms.

1,0297 lines. (There's no empty line.) : 121, 110, 123, 121, 167 ms

(I got an average of 1000ms for about 100,000 lines.)

Is this what you want?

Reply:
> If this behaves like other extension does, it's OK with me.

I'm checking by taking a picture of 'console.log', but whenever the text editor is modified, the function is executed.
I think this is the right practice, but there are concerns about performance...

Reply:
We tried to provide a code lens for the attribute when pressing the code lens at the top of the tools later.
However, when it was 10,000 lines, the performance deterioration was confirmed....

Reply:
> Is this what you want?

Yes seems so :) thanks for the measurement !

Reply:
>  but there are concerns about performance...

If you can get better performance code in short time, it'll be nice.
If not, adding a `TODO tune performance` in the code will do for now :)

Reply:
I got similar results when it was 10,000.
But I still have concerns about the performance, so I'll add it.